src/channel/NodeChannel.js

```diff
@@ -14,16 +14,15 @@ export default class NodeChannel extends Channel {
     /**
      * @internal
      * @param {string} address
-     * @param {number=} maxExecutionTime
+     * @param {number=} grpcDeadline
      */
-    constructor(address, maxExecutionTime) {
-        super();
+    constructor(address, grpcDeadline) {
+        super(grpcDeadline);
 
```

Comment:
Could potentially break some of our users. Sorry 😂 We still have the setter and the getter in this node channel but we don't use it anymore. Maybe we can make it backwards compatible and check which one is set and get the bigger number wdyt?

Reply:
Also we can add that we are deprecating this method, add a console.log for it and then mention it in the discussion about the v3 of the SDK?